org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterIasMovement.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.zigbee.internal.converter;
+
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.eclipse.smarthome.core.thing.binding.builder.ChannelBuilder;
+import org.openhab.binding.zigbee.ZigBeeBindingConstants;
+
+import com.zsmartsystems.zigbee.ZigBeeEndpoint;
+import com.zsmartsystems.zigbee.zcl.clusters.iaszone.ZoneTypeEnum;
+
+/**
+ * Converter for the IAS movement sensor.
+ *
+ * @author Henning Sudbrock - Initial Contribution
+ */
+public class ZigBeeConverterIasMovement extends ZigBeeConverterIas {
+
+    @Override
+    public boolean initializeConverter() {
+        bitTest = CIE_ALARM1;
+        return super.initializeConverter();
+    }
+
+    @Override
+    public Channel getChannel(ThingUID thingUID, ZigBeeEndpoint endpoint) {
+        if (!supportsIasChannel(endpoint, ZoneTypeEnum.VIBRATION_MOVEMENT_SENSOR)) {
```

Comment:
Just out of curiosity: Both of your two converters: `ZigBeeConverterIasMovement` and `ZigBeeConverterIasVibration` have the same condition, so devices will always get both channels. is this intended or does one of the converters use a wrong enum type?

Reply:
It would be nice to do better here, but I do not see how one can do better, due to how the IAS zone cluster works: The `zone state` of a device implementing the cluster is a bitmap, where bits `0` and `1` provide two alarm flags. Based on the `zone type` provided by the device, one knows what the two alarms mean (in this case, vibration and movement). But one cannot see which of the two alarms are actually implemented by a device (a device could only use the first, the second, or both alarm flags, i.e., here, depending on whether the device supports vibration alarms only, movement alarms only, or both).

Reply:
Thanks for the clarification!